application/views/js_init/js_keyboard.php

```diff
@@ -98,45 +98,45 @@
 });
 
 //select
-$(document).bind('keydown', 'o', read_message =  function(){  
+$(document).on('keydown', null, 'o', read_message =  function(){  
     $("#message_holder").children(":eq("+current_select+")").children('.message_container').find('div.message_content').toggle();
     $("#message_holder").children(":eq("+current_select+")").children('.message_container').find('span.message_preview').toggle();
     $("#message_holder").children(":eq("+current_select+")").children('.message_container').find('div.optionmenu').toggle();
     return false;
 });
 
-$(document).bind('keydown', 'd', function(){  
+$(document).on('keydown', null, 'd', function(){  
     $("#message_holder").children(":eq("+current_select+")").children('.message_container').find('div.detail_area').toggle();
 });
 
-$(document).bind('keydown', 'u', function(){  
+$(document).on('keydown', null, 'u', function(){  
     var dest = $('#back_threadlist').attr('href');
     document.location = dest;  
 });
 
-$(document).bind('keydown', 'x', function(){  
-    if($("#message_holder").children(":eq("+current_select+")").children('.message_container').find('.message_header').children('input.select_message').attr('checked')==true)
+$(document).on('keydown', null, 'x', function(){  
+    if($("#message_holder").children(":eq("+current_select+")").children('.message_container').find('.message_header').children('input.select_message').prop('checked')==true)
     {
-        $("#message_holder").children(":eq("+current_select+")").children('.message_container').find('.message_header').children('input.select_message').removeAttr('checked');
+        $("#message_holder").children(":eq("+current_select+")").children('.message_container').find('.message_header').children('input.select_message').prop('checked', false);
```

Comment:
Maybe `removeProp` would be more appropriate? Ref https://www.w3schools.com/tags/att_input_checked.asp

Reply:
Not required. https://api.jquery.com/removeprop/